Models/Map/Map.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using TheBattleShipClient.Models.Weapons;
+using TheBattleShipClient.Models.Ships;
+
+namespace TheBattleShipClient.Models
+{
+    public class Map
+    {
+        public string UserId { get; set; }
```

Comment:
Kam saugomas userId ir roomId? API to reikia, kad viską surišti tačiau client pusėje nematau kam reikėtų (userId apskritai net neimanoma gauti - ji zino tik serveris).
Dėl tos pačios priežasties kaip ir nereikalingas Room

Reply:
Nesigilindama sudejau pagal api o ateity kai naudosim atmesim - taip nutariau. Nes pradedama nezinojau, ko is referencu reiks, ko ne